release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,77 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-config {
+    description
+      "Configuration data for the netconf ptotocol";
+
+    leaf ssh {
+      type boolean;
+      default true;
+      description
+        "Enables the netconf protocol. The netconf is enabled by
+        default";
+    }
+
+    leaf port {
+      type uint16;
+      default 830;
+      description
+        "TCP port on which the netconf should listen(rfc6242)";
+    }
+
+    leaf session-absolute-timeout {
+      type uint32;
+      units minutes;
```

Comment:
Would suggest making this `seconds` since there are implementations that already support this

Reply:
comments incorporated . pls review